plugins/UM3NetworkPrinting/NetworkClusterPrinterOutputDevice.py

```diff
@@ -475,6 +476,38 @@ def __filterOurPrintJobs(self, print_jobs):
         username = self.__get_username()
         return [print_job for print_job in print_jobs if print_job["owner"] == username]
 
+    def _notifyConfigurationChangeRequired(self, old_print_jobs, new_print_jobs):
+        if old_print_jobs is None:
+            return
+
+        old_change_required_print_jobs = self.__filterConfigChangePrintJobs(self.__filterOurPrintJobs(old_print_jobs))
+        new_change_required_print_jobs = self.__filterConfigChangePrintJobs(self.__filterOurPrintJobs(new_print_jobs))
+        old_change_required_print_job_uuids = set([pj["uuid"] for pj in old_change_required_print_jobs])
+
+        for print_job in new_change_required_print_jobs:
+            if print_job["uuid"] not in old_change_required_print_job_uuids:
+
+                printer_name = self.__getPrinterNameFromUuid(print_job["assigned_to"])
+                if printer_name is None:
+                    printer_name = i18n_catalog.i18nc("@info:status", "Unknown printer")
```

Comment:
Is this in the translation catalog already?

Reply:
No... I'll try to find a matching string

Reply:
It's going to be: msgctxt "@label"
msgid "Unknown"